backend/src/services/tba.py

```diff
@@ -115,3 +116,16 @@ async def fetch_matches(self, year: int) -> int:
             raise
 
         return len(matches)
+
+    async def fetch_teams_events(self, year: int) -> int:
+        matches_query = """
+        INSERT INTO team_events (team_key, event_code)
```

Comment:
```suggestion
        INSERT INTO team_events (team_key, event_key)
```

same change for `event_key` + incomplete query